docs/mint.json

```diff
@@ -123,7 +123,14 @@
               "group": "Configu",
               "pages": [
                 "integrations/store/configu/overview",
-                "integrations/store/configu/config-explorer",
+                {
+                  "group": "Config Explorer",
+                  "pages": [
+                    "integrations/store/configu/config-explorer",
+                    "integrations/store/configu/config-explorer/upsert-configs-wizard",
+                    "integrations/store/configu/config-explorer/config-set-exporter"
```

Comment:
Keep in mind that this change requires an update to the SaaS references to these sections.